nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4087,35 +4140,90 @@ async fn image_list(
 /// Create a new image in a project.
 #[endpoint {
     method = POST,
-    path = "/organizations/{organization_name}/projects/{project_name}/images",
+    path = "/v1/images",
     tags = ["images"]
 }]
+async fn image_create_v1(
+    rqctx: RequestContext<Arc<ServerContext>>,
+    query_params: Query<params::ProjectSelector>,
+    new_image: TypedBody<params::ImageCreate>,
```

Comment:
Since `ImageCreate` doesn't have an `is_global` param, that means you can't create an image already promoted. Worth considering?

Reply:
It's intentional. Promoting an image is an action that requires different permissions than creating an image in a project. I wanted to avoid conditional permissions based on the presence of a given property.